### ISSUE TITLE
Guys stop creating version requests!

### ISSUE THREAD

**Issue by miqumi** [ddf8196/BetterRenderDragon#50]
Please, guys;
This is a request for you all those who are posting issues on the github page which are really nonsensical as they are not issues at all!
Stop posting issues begging for new versions.
Porting BRD to new versions takes hell lotta time and brainpower

If you do NOT believe me, try porting any old version of brd to one or two versions ahead; if it takes ANY more than 4 weeks know that the BRD devs are doing a great job and start encouraging them.

###### _I do know the fact that most of you are illiterate with C++ so if you ARE then you dont even need to read the part above ^^-_

TL;DR : Don't make an issue about the latest version not working. It doesn't have automagic.

**Comment by Th3E0ne** [comment]
There is a 1.20.32 working version of BRD on the YSS Discord Server that hasn't been released on Github yet.

**Comment by miqumi** [comment]
> There is a 1.20.32 working version of BRD on the YSS Discord Server that hasn't been released on Github yet.

unrelated but the latest release on github should also be working on .32

**Comment by Arctrometa** [comment]
We'd appreciate it if everyone would abide by this.
We have new versions of BetterRD in development and a non beta version of BetterRD 1.3.5 with added support for the newest previews and release versions is available on the YSS discord.

**Comment by ENDERMANYK** [comment]
just get the dll file from [here](https://github.com/ddf8196/BetterRenderDragon/actions/runs/6559552473) and replace the same name file under dlls folder.

**Comment by miqumi** [comment]
> just get the dll file from [here](https://github.com/ddf8196/BetterRenderDragon/actions/runs/6559552473) and replace the same name file under dlls folder.

right
!

**Comment by miqumi** [comment]
> We'd appreciate it if everyone would abide by this. We have new versions of BetterRD in development and a non beta version of BetterRD 1.3.5 with added support for the newest previews and release versions is available on the YSS discord.

👍

**Comment by Th3E0ne** [comment]
It is completely related :)

**Comment by Arctrometa** [comment]
Idk man, seems pretty open source to me.

> Can I ask when it will become open-source yet again?

**Comment by notchyves** [comment]
Oh I meant open source for the latest version. 1.20.41

**Comment by Arctrometa** [comment]
1.20.41 support has been added to the src for a while now.

**Comment by miqumi** [comment]
> Oh I meant open source for the latest version. 1.20.41

...
The code is updated by da men as soon as they complete support for a new version
So yeah if there is a new version support in the src you can build yourself
its a really smol dll so you shouldn't be waiting for long build times



_or be smart and download from github actions_
